plugins/azure/virtualmachines/vmVTPMEnabled.js

```diff
@@ -0,0 +1,48 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'VM vTPM Enabled',
+    category: 'Virtual Machines',
+    domain: 'Compute',
+    description: 'Ensure Virtual Trusted Platform Module (vTPM) is enabled for Azure virtual machines (VM).',
+    more_info: 'vTPM is TPM2.0 compliant and enhances security by validating VM boot integrity and providing a secure storage mechanism for keys and secrets.',
```

Comment:
```suggestion
    more_info: 'vTPM is TPM2.0 compliant and enhances security by validating VM boot integrity and providing a secure storage mechanism for keys and secrets.',
```
add a little more info in here